tests/snapshots/tests__standard@generic-for-multiline.lua.snap

```diff
@@ -0,0 +1,63 @@
+---
+source: tests/tests.rs
+expression: format(&contents)
+
+---
+local function system(world)
+	for id, model, lasering, transform in world:query(
+		Components.Model,
+		Components.Lasering,
+		Components.Transform,
+		Components.Mothership
+	) do
+	end
+end
+
+local function system(world)
+	for id,
+		model,
+		lasering,
+		transform,
+		id,
+		model,
+		lasering,
+		transform,
+		id,
+		model,
+		lasering,
+		transform,
+		id,
+		model,
+		lasering,
+		transform
+	in world:query(Components.Model, Components.Lasering, Components.Transform, Components.Mothership) do
+	end
+end
+
+local function system(world)
+	for id,
```

Comment:
why isn't `id` indented on the next line like the rest of the arguments? again, I'm thinking of consistency with the `if` and `else` cases

Reply:
I don't think there was a particular reason for this, just how I did it (its interesting how the the `id` in the first line aligns nicely with the other variables too).

Again, I agree its better to be consistent here so I'll change it